geps/gep-2162.md

```diff
@@ -0,0 +1,180 @@
+# GEP-2162: Supported features in GatewayClass Status
+
+* Issue: [#2162](https://github.com/kubernetes-sigs/gateway-api/issues/2162)
+* Status: Provisional
+
+## TLDR
+
+This GEP proposes to enhance the [GatewayClassStatus](https://github.com/kubernetes-sigs/gateway-api/blob/f2cd9bb92b4ff392416c40d6148ff7f76b30e649/apis/v1beta1/gatewayclass_types.go#L185) to include a list of Gateway API features supported by the installed GatewayClass. 
+
+## Goals
+
+* Improve UX by enabling users to easily see what features the implementation (GatewayClass) support.
+
+* Standardize features and conformance tests names.
+
+* Automatically run conformance tests based on the supported features populated in GatewayClass status.
+
+* Provide foundation for tools to block or warn when unsupported features are used.
+
+
+## Non-Goals
+
+* Validate correctness of supported features published by the implementation. Meaning we don't intend to verify whether the supported features reported by the implementation are indeed supported.
```

Comment:
Non-blocking: Maybe could add a note that it should make it very easy for any individual to verify by running conformance tests against the GatewayClass.

Reply:
Yep, if you do this one, I will LGTM and this can go in.